cfme/common/__init__.py

```diff
@@ -238,16 +241,37 @@ def step(self, *args, **kwargs):
         self.prerequisite_view.toolbar.policy.item_select('Manage Policies')
 
 
+class AssignedTags(ParametrizedView):
+    PARAMETERS = ("tag",)
+    ALL_TAGS = ".//a[contains(@class, 'pf-remove-button')]"
+    tag = Text(ParametrizedLocator(
```

Comment:
Why a ParametrizedLocator here? There's no parameter in the locator that I see?

Reply:
there should be... I missed it and now added in the correct place.

Reply:
fixed